bin/i18n/sync-in.rb

```diff
@@ -556,6 +557,40 @@ def write_to_yml(type, strings)
   end
 end
 
+# Merging dashboard/config/courses/*.course with courses.yml
+# These files contain resources used by UnitGroup (used in the landing pages of full courses).
+# https://studio.code.org/courses/csd-2021
+# All other resources used in Unit come from scrip_json files (used in the landing pages for each Unit).
+# https://studio.code.org/s/csd1-2021
+def localize_course_resources
+  puts "Preparing course resources"
+
+  # Currently, csd is the only fully translatable course that has resources in this directory
+  translatable_courses = %w(csd)
+  resources = {}
+  Dir.glob(File.join('dashboard', 'config', 'courses', '*.course')).sort.each do |file|
```

Comment:
Do you need `.sort` here? Aren't you just iterating over all of the `*.course` files in `dashboard/config/courses`?

Reply:
I don't think it is necesary

Reply:
I agree 👍. Using sort here just sorts the array before you iterate over it, only adding time complexity.